Prac8/unreliable_car.py

```diff
@@ -0,0 +1,15 @@
+from Prac7.Car import Car
+import random
```

Comment:
could also use: from random import randint
so that you don't need to reference random later, however this isn't required (up to you).